stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.14.2
+version: 1.14.3
```

Comment:
Since this adds a feature, it should be the minor version number that is incremented.